sigar/src/test/java/io/crate/monitor/SigarExtendedNodeInfoTest.java

```diff
@@ -95,7 +99,12 @@ public void testOsStats() throws Exception {
         assertThat(stats.timestamp(), greaterThan(0L));
         assertThat(stats.uptime().millis(), greaterThan(0L));
         ExtendedOsStats.Cpu cpu = stats.cpu();
+        ExtendedOsStats.CgroupMem cgroupMem = stats.cgroupMem();
         assertThat(cpu.sys(), greaterThan((short) -1));
+        if (Constants.LINUX && cgroupMem != null) {
```

Comment:
can cgroupMem be null if Constants.LINUX ? Here: https://github.com/crate/crate/pull/6596/files#diff-c3a89be5cb1da980dffca5179ed2bb33R227 we don't have such check.